osler/inventory/views.py

```diff
@@ -20,10 +24,15 @@ def get_queryset(self):
                     select_related('unit').\
                     select_related('category').\
                     select_related('manufacturer').\
-                    order_by('category', 'name').\
+                    order_by('category', 'name', 'dose', 'expiration_date').\
                     exclude(stock=0).all()
         return druglist
 
+    def get_context_data(self, **kwargs):
```

Comment:
great use of querysets both here in this function as well as in `get_queryset` above!

Reply:
Thank you!